exec/java-exec/src/test/java/org/apache/drill/exec/record/TestRecordBatchSizer.java

```diff
@@ -45,7 +45,7 @@
   private void verifyColumnValues(ColumnSize column, int stdDataSizePerEntry, int stdNetSizePerEntry,
                                   int dataSizePerEntry, int netSizePerEntry, int totalDataSize,
                                   int totalNetSize, int valueCount, int elementCount,
-                                  int estElementCountPerArray, boolean isVariableWidth) {
+                                  int cardinality, boolean isVariableWidth) {
```

Comment:
Maybe explain the cardinality here since I've seen it (or have used it) in three different ways:

* Type cardinality: (Required: 1, Optional: (0, 1), Repeated: *)
* Batch cardinality: the number of rows in a batch. (Or, equivalently, the number of top-level values in a vector.)
* Array cardinality: the number of values in an array (a single top-level value which is repeated)

"Cardinality" is a good word, may just need disambiguation.